### DISCUSSION TITLE
Top menu always visible in file list

### DISCUSSION THREAD

**Discussion by Chris78c** [music-assistant/support#2571] [Feature requests and ideas]
when scrolling through the media list, I find it useful to have the top menu (marked in red in the video) always in view when scrolling down, so I can use it even if I'm way down the list (both from App-Mobile and Browser). Or alternatively you could make it pop-up, when you scroll up the menu appears, when you scroll down the menu disappears.
Thank you very much for your attention.

https://github.com/music-assistant/hass-music-assistant/assets/165342443/c51626f9-06b8-4a41-942a-c40b5a66aeb2

**Comment by uSlackr** [comment]
Jumping in to add my two cents.  Not only does the bar disappear, it takes the search box with it.  So if you decide to stop scrolling, you have to scroll back to the top to search.
